packages/react/react/src/components/Portal/Portal.tsx

```diff
@@ -0,0 +1,22 @@
+import ReactDOM from 'react-dom';
+import { useIsMounted } from '../../hooks/useIsMounted';
+
+interface Props {
+  children: React.ReactNode;
+  id: string;
+}
+
+/** @tossdocs-ignore */
+export function Portal({ id, children }: Props) {
+  const isMount = useIsMounted();
+
+  /**
+   * With this code, it is possible to solve the "window is not defined" and "Hydration Error" that can occur in SSR.
+   */
+  if (!isMount) {
+    return <></>;
+  }
```

Comment:
```suggestion
  const isMounted = useIsMounted();

  /**
   * With this code, it is possible to solve the "window is not defined" and "Hydration Error" that can occur in SSR.
   */
  if (!isMounted) {
    return <></>;
  }
```

I think this is more appropriate😄

### Usage

![image](https://github.com/toss/slash/assets/69495129/cb8b9d1c-2f84-4267-8980-8d7afcf211b8)

Reply:
That's something I missed..!!!

Thank you for the feedback. I agree with your opinion 👍

I'll fix that code in a moment🙆‍♂️